.vscode/configurationCache.log

```diff
@@ -0,0 +1 @@
+{"buildTargets":[],"launchTargets":[],"customConfigurationProvider":{"workspaceBrowse":{"browsePath":[]},"fileIndex":[]}}
```

Comment:
Please ignore files in `.vscode`, they are not related to the project.

I recommend adding them into ` ~/.gitignore_global`